src/Sylius/Bundle/ApiBundle/Command/Cart/AssignCartToUser.php

```diff
@@ -13,7 +13,7 @@
 
 namespace Sylius\Bundle\ApiBundle\Command\Cart;
 
-class BlameCart
+readonly class AssignCartToUser
```

Comment:
Why did you change the name? In my opinion it should remain the previous one, it is clear and we have been using a similar term in the backend for a long time

Reply:
The term Blame is vague, at least to me, so I had to open the class to find out what it actually does.

I also asked our artificial friend for their opinion:
<img width="792" alt="image" src="https://github.com/user-attachments/assets/ad10953f-fccf-4d47-a306-84e4c18448c2">

I can revert these changes, but in my opinion, that proposal isn't that bad 💃

Reply:
I understand that this may not be fully clear, nevertheless, such wording has existed in Sylius for a long time, including in the context of the UI, and what's more, this should not be part of this quite huge PR, so from my side, undo this change here 🖖🏻